File: backend/src/main/java/com/x/backend/pojo/common/Account.java
package com.x.backend.pojo.common;

import lombok.Data;

import java.util.Date;

@Data
public class Account {
    private Integer aId;
    private String username;
    private String password;
    private String email;
    private String role;
    private String vip;
    private Date createdAt; // 创建时间
    private String inviteCode; // 邀请码

    // 是否封禁
    private Boolean isBanned; // true:封禁 false:未封禁

    private Date lastLoginTime;
}

File: backend/src/main/java/com/x/backend/pojo/common/Account.kt
package com.x.backend.pojo.common

import lombok.Data
import lombok.Getter
import lombok.Setter
import java.util.*

@Data
@Getter
@Setter
open class Account {
    var aId: Int? = null
    var username: String? = null
    var password: String? = null
    var email: String? = null
    var role: String? = null
    var vip: String? = null
    var createdAt: Date? = null // 创建时间
    var inviteCode: String? = null // 邀请码

    // 是否封禁
    var isBanned: Boolean? = null // true:封禁 false:未封禁
}
